pasha_modules/util.coffee

```diff
@@ -255,7 +233,6 @@ module.exports = {
     sendConfirmEmail: sendConfirmEmail
     sendStatusEmail: sendStatusEmail
     pagerdutyAlert: pagerdutyAlert
-    startNag: startNag
```

Comment:
that slack link does not work for me ;)

On Wed, Feb 17, 2016 at 8:56 AM Bence Kiglics <notifications@github.com>
wrote:

> In pasha_modules/util.coffee
> <https://github.com/prezi/pasha/pull/17#discussion_r53135358>:
>
> > @@ -255,7 +233,6 @@ module.exports = {
> >      sendConfirmEmail: sendConfirmEmail
> >      sendStatusEmail: sendStatusEmail
> >      pagerdutyAlert: pagerdutyAlert
> > -    startNag: startNag
>
> Btw. we might want to refactor this file in the future to use @ names for
> public functions
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prezi/pasha/pull/17/files#r53135358>.
>